### PR TITLE
Remove kernel version requirement on digitalocean

### PR DESCRIPTION
**Pull request by siddharthist**
Tested on DO, deploys sucessfully

Unrelatedly, this also wraps the line of the ansible version requirement.

### CODE REVIEW

File: addons/glusterfs.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@
 # CHECK SECURITY - when customizing you should leave this in. If you take it out
 # and forget to specify security.yml, security could be turned off on components
 # in your cluster!
-- include: playbooks/check-requirements.yml
+- include: "{{ playbook_dir }}/check-requirements.yml"
```
Comment (ryane) [inline addons/glusterfs.yml]: this still results in a failure:

```
$ ansible-playbook -e @security.yml addons/glusterfs.yml
ERROR: file could not read: /home/ryan/Projects/microservices-infrastructure/addons/check-requirements.yml
```

**Comment by ryane** [comment]
@TanyaCouture @sehqlr this is ready for testing again. Need to verify that a Digital Ocean deployment is successful and that it is possible to install the glusterfs addon. Also, should verify that you can run the reboot-hosts playbook. Make sure you are using a terraform.yml with the latest changes from terraform.sample.yml

**Comment by sehqlr** [comment]
Tests completed successfully. We are a go

**Comment by sehqlr** [comment]
Since we are starting to follow the workflow in #957, is it enough that I tested it, or does someone else need to sign off on it too? I know that this needs to be merged with release06 and master.

**Comment by ryane** [comment]
lgtm

**Pull request merged**